src/compiler/crystal/codegen/fun.cr

```diff
@@ -101,9 +101,13 @@ class Crystal::CodeGenVisitor
           context.fun.add_attribute LLVM::Attribute::UWTable, value: @program.has_flag?("aarch64") ? LLVM::UWTableKind::Sync : LLVM::UWTableKind::Async
         {% end %}
 
-        if @program.has_flag?("darwin")
+        if @frame_pointers == FramePointers::Always
```

Comment:
**suggestion:** `Enum`'s predicate methods are more concise and type safe:
```suggestion
        if @frame_pointers.always?
```

Reply:
ngl I forgot those were a thing, does look way better